chain/cosmos/chain_node.go

```diff
@@ -380,10 +380,19 @@ func (tn *ChainNode) FindTxs(ctx context.Context, height uint64) ([]blockdb.Tx,
 // with the chain node binary.
 func (tn *ChainNode) TxCommand(keyName string, command ...string) []string {
 	command = append([]string{"tx"}, command...)
+	var gasPrices, gasAdjustment string = tn.Chain.Config().GasPrices, fmt.Sprint(tn.Chain.Config().GasAdjustment)
+	for i := 0; i < len(command); i++ {
```

Comment:
With the most recent modification, it will potentially add the flags twice. I am not sure if it supports this. I was suggesting we have bools such as `gasPricesFound, gasAdjustmentFound := false, false` then in the for loop, if we find them, set the relevant bool to true. Then after the for loop, check each bool and append the chain config value for false

Reply:
Ahh!
Third times the charm? @agouin